apps/web-tools/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@ledgerhq/web-tools",
+  "version": "0.1.0",
+  "private": true,
+  "browser": {
+    "fs": false,
+    "tls": false,
+    "net": false,
+    "dns": false
+  },
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@ledgerhq/coin-framework": "workspace:^",
+    "@ledgerhq/domain-service": "workspace:^",
+    "@ledgerhq/errors": "workspace:^",
+    "@ledgerhq/hw-app-eth": "workspace:^",
+    "@ledgerhq/hw-transport": "workspace:^",
+    "@ledgerhq/hw-transport-http": "workspace:^",
+    "@ledgerhq/hw-transport-web-ble": "workspace:^",
+    "@ledgerhq/hw-transport-webhid": "workspace:^",
+    "@ledgerhq/hw-transport-webusb": "workspace:^",
+    "@ledgerhq/live-common": "workspace:^",
```

Comment:
Aren't `coin-*` libs required too? To support algorand + polkadot + evm that have already been extracted?

Reply:
( same as in common-tools ) nope live-common is still a monolith that aggregate the coin together (so what you said is done at the live-common level)

Reply:
Ah true, no direct dependency